src/coreclr/jit/emitxarch.cpp

```diff
@@ -128,6 +139,122 @@ bool emitter::IsAVXInstruction(instruction ins) const
     return UseVEXEncoding() && IsSSEOrAVXInstruction(ins);
 }
 
+bool emitter::IsAVX512Instruction(instruction ins) const
+{
+    switch (ins)
+    {
+        // No EVEX Encoding exists at all.
```

Comment:
This might be another place where a flag in the instr table would be good.

Reply:
Added a ToDO comment for now